demo/variable_definitions/read_variable_definitions.ipynb

```diff
@@ -0,0 +1,375 @@
+    "1. Clone this repo `git clone https://github.com/statisticsnorway/dapla-toolbelt-metadata.git`\n",
+    "1. Install the packages `poetry install; poetry shell`\n",
+    "1. Open this notebook\n",
+    "1. Choose the correct kernel\n",
+    "1. Run this notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from typing import Any\n",
+    "\n",
+    "import requests\n",
+    "from rich import print_json\n",
+    "\n",
+    "token = os.environ.get(\"OIDC_TOKEN\")\n",
+    "active_group = os.environ.get(\"DAPLA_GROUP_CONTEXT\")\n",
+    "\n",
+    "if not token:\n",
+    "    msg = (\n",
+    "        \"\"\"Environment variable 'OIDC_TOKEN' not set.\n",
+    "If you are running on Dapla Lab, restart your service with Buckets access configured.\n",
+    "If you are not running on Dapla Lab, obtain the value for it by:\n",
+    "    1. Launch a service (Jupyterlab or VSCode) at https://lab.dapla.ssb.no\n",
+    "    2. Run command 'echo $OIDC_TOKEN'\n",
+    "    3. Run 'export OIDC_TOKEN=<value obtained from Dapla Lab>' on this machine.\n",
+    "\"\"\"\n",
+    "    )\n",
+    "    raise OSError(msg)\n",
+    "\n",
+    "if not active_group:\n",
+    "    msg = (\"DAPLA_GROUP_CONTEXT environment variable not set, please set it to the name of the Dapla group you wish to act as.\")\n",
+    "    raise OSError(msg)\n",
+    "\n",
+    "print(f\"Your Dapla group context is: {active_group}\")\n",
+    "\n",
+    "# Only available from Statistics Norway's networks, requires authorization\n",
+    "TEST_INTERNAL_SERVER = \"https://metadata.intern.test.ssb.no\"\n",
+    "\n",
+    "def vardef_request(path: str, method: str=\"GET\", body: dict | None = None, params: dict | None = None) -> Any:\n",
+    "    \"\"\"Make a request to vardef and return the json content from the response.\"\"\"\n",
+    "    if not params:\n",
+    "        params = {}\n",
+    "\n",
+    "    params.update({\"active_group\": active_group})\n",
+    "\n",
+    "    response = requests.request(\n",
+    "        method,\n",
+    "        f\"{TEST_INTERNAL_SERVER}{path}\",\n",
+    "        params=params,\n",
+    "        headers={\"Authorization\": f\"Bearer {token}\"},\n",
+    "        json=body,\n",
+    "        timeout=10,\n",
+    "        )\n",
+    "    response.raise_for_status()\n",
+    "    return response.json()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Drafts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a draft"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "body = {\n",
```

Comment:
Add one comment at top. Something like: All values can be edited, but if use of example remember that short_name must be unique... and maybe that this can return 409 Conflict.